.gitignore

```diff
@@ -25,4 +25,7 @@ SYSTEM_RUNNING
 
 # One of the functional tests puts a file in the static directory that is very large.
 # we don't want to add this to our repo.
-src/test/webapp/static/largefile.txt
\ No newline at end of file
+src/test/webapp/static/largefile.txt
+*.prefs
```

Comment:
Would you document this section, like, "project configuration files for Eclipse" or something.